mlir-sycl/include/mlir/Dialect/SYCL/IR/SYCLHostOps.td

```diff
@@ -37,10 +37,10 @@ def SYCLHostHandlerOp : SYCLOpTrait<"SYCLHostHandlerOp">;
 
 def SYCLHostConstructorOp : SYCL_HostOp<"constructor",
     [MemoryEffectsOpInterface]> {
-  let summary = [{
-    Operation representing the construction of a SYCL object, i.e., allocation,
-    and member initialization.
+  let summary = "Represents the construction of a SYCL object, " #
+                "i.e., allocation, and member initialization.";
```

Comment:
Should we mention allocation here? Allocation happens previously, the operation takes the pointer.

Reply:
Will drop.